src/elements/content-sidebar/activity-feed/task-new/TaskCompletionRuleIcon.js

```diff
@@ -15,11 +18,14 @@ type Props = {|
 
 const TaskCompletionRuleIcon = ({ completionRule }: Props): React.Node =>
     completionRule === TASK_COMPLETION_RULE_ANY && (
-        <Tooltip position="top-center" text={<FormattedMessage {...messages.taskAnyAffordanceTooltip} />}>
-            <span>
-                <IconAnyTask height={11} width={11} className="bcs-TaskCompletionRuleIcon" />
-            </span>
-        </Tooltip>
+        <span className="bcs-TaskCompletionRuleIcon">
+            <Tooltip position="top-center" text={<FormattedMessage {...messages.taskAnyAffordanceTooltip} />}>
+                <LabelPill.Pill>
+                    <LabelPill.Icon Component={Avatar16} />
+                    <span>1</span>
```

Comment:
What is this meant to represent? Should it be wrapped in `LabelPill.Text`?

Reply:
It represents a number 1 next to the Avatar icon. It isn't wrapped in LabelPill.Text bc it then inherits the LabelPill css properties and is hard to work w/ to match the designs for the icon. Wrapping it in a span places it right next to the icon and is easier to adjust for design purposes

Reply:
Forgot to add that. Thanks for pointing it out.